recipes/textx/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "textx" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/textX-{{ version }}.tar.gz
+  sha256: cd432adb82e348960cecdcabe68dbb916fe6eb2e4cb067c4a4b68b8c9ec51acf
+
+build:
+  entry_points:
+    - textx = textx.cli:textx
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - arpeggio >=2.0.0
+    - click >=7.0,<8.0
+    - future
+    - python >=3.6
+
+test:
+  source_files:
+    - tests
+  imports:
+    - textx
+  commands:
+    - pip check
+    - textx --help
+    # This test file is causing problem can't easily exclude by pytest commands
+    - rm tests/functional/test_objcrossref_positions.py
+    - pip install -e  --no-deps tests/functional/subcommands/example_project
+    - pip install -e --no-deps tests/functional/registration/projects/types_dsl
+    - pip install -e --no-deps tests/functional/registration/projects/data_dsl
+    - pip install -e --no-deps tests/functional/registration/projects/flow_dsl
+    - pip install -e --no-deps tests/functional/registration/projects/flow_codegen
```

I guess this is the correct incarnation for this.

```suggestion
    - pip install -e tests/functional/subcommands/example_project --no-deps
    - pip install -e tests/functional/registration/projects/types_dsl --no-deps
    - pip install -e tests/functional/registration/projects/data_dsl --no-deps
    - pip install -e tests/functional/registration/projects/flow_dsl --no-deps
    - pip install -e tests/functional/registration/projects/flow_codegen --no-deps
```